test/node-api/test_worker_terminate_finalization/test.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 const common = require('../../common');
 
+// TODO(addaleax): Run this test once it stops failing under ASAN/valgrind.
+// Refs: https://github.com/nodejs/node/issues/34731
```

Comment:
Optional non-blocking suggestion: Should the comment mention https://github.com/nodejs/node/issues/35778 as well? And maybe https://github.com/nodejs/node/pull/35777 (this pull request) too?